lib/vrl/compiler/src/expression.rs

```diff
@@ -103,16 +122,24 @@ clone_trait_object!(Expression);
 
 #[derive(Debug, Clone, PartialEq)]
 pub enum Expr {
+    #[cfg(feature = "expr-literal")]
     Literal(Literal),
     Container(Container),
```

Comment:
I haven't added support for container-types yet. I'll probably do so in a follow-up PR.